2.6/examples/replica/README.md

```diff
@@ -0,0 +1,84 @@
+# MongoDB Replica Set example
+
+**WARNING: This is only a Proof-Of-Concept example and it is not meant to be used in
+production. Use at your own risk.**
+
+## What is a MongoDB replica set?
+
+A cluster of MongoDB servers that implements master-slave replication and automated failover.
+MongoDB’s recommended replication strategy.
+
+## Deployment
+
+To create a MongoDB replica set in OpenShift v3, you can use the template
+included in this repository and create a new deployment right away:
+
+```
+$ oc new-app 2.6/examples/replica/mongodb-clustered.json
+```
+
+Since the provided template uses ephemeral storage, after every redeploy you get
+an empty database, and **all data from a previous deploy is lost**.
+
+## How does it work?
+
+### Service 'mongodb'
+
+This resource defines a [headless service](http://kubernetes.io/v1.0/docs/user-guide/services.html#headless-services)
+that serves as an entry point to the replica set. The service endpoints are
+the pods created by the 'mongodb' DeploymentConfig.
+
+
```

Comment:
This is almost a copy-paste from `2.4/examples/replica/README.md`... except for 3 extra empty lines...
Here is one.